### DISCUSSION TITLE
Allow different SMTP servers in flows

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#19445] [Feature Requests]
### Summary

The SMTP server is configured in .env and therefore applies to all emails sent.
To optimize mass mailings, it would be good to replace the default SMTP server in some flows

### Basic Example

Ability to use other
# EMAIL_MAILGUN_API_KEY="key-1234123412341234"
# EMAIL_MAILGUN_DOMAIN="a domain name from https://app.mailgun.com/app/sending/domains"
during sending from a flow

### Motivation

This could allow easier accounting per vendor

### Detailed Design

set 
EMAIL_MAILGUN_API_KEY
EMAIL_MAILGUN_DOMAIN
programmatically

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

none

### Alternatives

-

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
